internal/services/paloalto/next_generation_firewall_vhub_local_rulestack_resource_test.go

```diff
@@ -129,6 +130,28 @@ func (r NextGenerationFirewallVWanResource) Exists(ctx context.Context, client *
 }
 
 func (r NextGenerationFirewallVWanResource) basic(data acceptance.TestData) string {
+	if !features.FivePointOhBeta() {
+		return fmt.Sprintf(`
+provider "azurerm" {
+  features {}
+}
+
+%[1]s
+
+resource "azurerm_palo_alto_next_generation_firewall_virtual_hub_local_rulestack" "test" {
+  name                = "acctest-ngfwvh-%[2]d"
+  resource_group_name = azurerm_resource_group.test.name
+  rulestack_id        = azurerm_palo_alto_local_rulestack.test.id
+  plan_id             = "panw-cngfw-payg"
+
+  network_profile {
+    virtual_hub_id               = azurerm_virtual_hub.test.id
+    network_virtual_appliance_id = azurerm_palo_alto_virtual_network_appliance.test.id
+    public_ip_address_ids        = [azurerm_public_ip.test.id]
+  }
+}
+`, r.template(data), data.RandomInteger)
+	}
```

Comment:
The basic test config shouldn't include optional arguments. I think this can be removed since we've added `plan_id` to the `complete` test configuration

Reply:
Given service team has deprecated the default value "panw-cloud-ngfw-payg", so we have to explicitly set it with new default value. Otherwise it would fail with below error.

Note: All related test cases on Teamcity are failed with below same error.

```
Error: creating Firewall (Subscription: "xxxxx"
Resource Group Name: "acctestRG-PANGFWVN-250116020412783950"
Firewall Name: "acctest-ngfwvn-250116020412783950"): performing CreateOrUpdate: unexpected status 400 (400 Bad Request) with error: PaymentRequired: SaaS Purchase Payment Check Failed as validationResponse was {"isEligible":false,"errorMessage":"Plan 'panw-cloud-ngfw-payg' is defined as stop sell in offer 'pan_swfw_cloud_ngfw'."}
```